Sasquatch/Sasquatch/Constants.h

```diff
@@ -36,3 +36,5 @@ static int const kMSStoragePageSize = SQLITE_DEFAULT_PAGE_SIZE;
 #else
 static int const kMSStoragePageSize = 4096;
 #endif
+
+static const NSString *kMSIntConfigUrl = @"https://mobilecentersdkdev.blob.core.windows.net";
```

Comment:
that's going away, the one you're looking for is `https://config-integration.dev.avalanch.es` you might want to copy the blobs there as part of this task to avoid breaking our test apps.